sample-android/lint_baseline.xml

```diff
@@ -1,6 +1,68 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="6" by="lint 8.5.0-alpha02" type="baseline" client="" dependencies="true" name="" variant="all" version="8.5.0-alpha02">
 
+    <issue
+        id="DuplicateDefinition"
```

Comment:
I think this duplicate definition comes due to a cache issue, could you regenerate this baseline after a clean build and killing workers? I will investigate this cache issue separately.

Reply:
sure

Reply:
same result